Automattic/studio
# Studio: Fix site starting too early before import

Related to https://github.com/Automattic/dotcom-forge/issues/8547


## Proposed Changes

This PR ensures that the site is started only after the import is finished.

## Testing Instructions



* Pull the changes from this branch
* Run `nvm use && npm install && STUDIO_IMPORT_EXPORT=true npm run start` to start Studio.
* Click on `Add site` in the sidebar and add an import file
* Start adding the site
* Observe that after the import is completed, the site is started and you can log into WP-Admin without being asked to log in
* Observe that the site thumbnail is immediately updated when the import is finished to a new theme

## Pre-merge Checklist
